compute/ARMComputeEx/src/runtime/NEON/functions/NEFullyConnectedLayerEx.cpp

```diff
@@ -160,13 +161,18 @@ void NEFullyConnectedLayerEx::configure_fc_fc(const ITensor *input, const ITenso
   configure_mm(input, weights, output);
 }
 
+const ITensor *t_output;
+const ITensor *t_input;
+const ITensor *t_weight;
```

Comment:
`const ITensor *t_output;`
`const ITensor *t_input;`
`const ITensor *t_weight;`
do we need this variables ?
I can't find any use of them

Reply:
Oops.. I missed it. Thanks.

Reply:
Removed.